implementations/rust/ockam/ockam_command/src/docs.rs

```diff
@@ -128,7 +128,11 @@ impl FencedCodeBlockHighlighter<'_> {
         }
     }
 
+    // TODO: fix the fenced code block highlighter (currently disabled), as it does not work on macOS or Linux
```

Comment:
completely remove the code rather than leave dead code, you can easily recover it anyway.

Reply:
Unfortunately, there are lots of depenendencies which will have to be re-created if I remove all the dead code. Instead I've removed the call to the broken function here. And put an explicit TODO at the call site, and at the broken function. This will also facilitate the community making a fix for this, when I create an issue for this.

Reply:
nice!